plumbing/api.go

```diff
@@ -120,7 +125,17 @@ func (api *API) BlockGet(ctx context.Context, id cid.Cid) (*types.Block, error)
 	return api.chain.GetBlock(ctx, id)
 }
 
-// MessagePoolPending lists messages in the pool.
+// DealsLs a slice of all storagedeals in the local datastore and possibly an error
+func (api *API) DealsLs() ([]*storagedeal.Deal, error) {
+	return api.storagedeals.Ls()
+}
+
+// DealPut puts a given deal in the datastore
+func (api *API) DealPut(storageDeal *storagedeal.Deal) error {
+	return api.storagedeals.Put(storageDeal)
+}
```

Comment:
I like where this is going. In the future, we could consider adding a `DealRemove` or `DealDelete` for letting users define their own data retention policies around deals.

We briefly discussed expiration policies around deals when we initially added deal persistence, but we never came up with a great solution for it. With this, we could potentially have an automatic delete after a few days for most users and let larger miners with capacity and interest in holding records for longer define their own policies through manual commands.